packages/render-fidelity-tools/test/renderers/model-viewer/index.html

```diff
@@ -60,7 +60,9 @@
         // the poster has completely faded away. 
```

Comment:
Just added more raf to see if the bug still exist.

Reply:
Does this fix the bug? Maybe we can make the number of RAF configurable